.env

```
@@ -0,0 +1,5 @@
+SECRET_KEY=django-insecure-development-key-change-in-production
+DEBUG=True
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DEBUG key should go before the SECRET_KEY key

(UnorderedKey)